app/controllers/v0/sessions_controller.rb

```diff
@@ -35,7 +35,7 @@ def saml_callback
 
     def persist_session_and_user!
       @session = Session.new(user_attributes.slice(:uuid))
-      @current_user = User.find(@session.uuid) || User.new(user_attributes)
+      @current_user = User.find(@session.uuid) || UserMviDelegate.new(User.new(user_attributes)).create
```

Comment:
@saneshark having an after_initialize on user to fetch mvi data was requiring lots of stubbing in specs, a code smell 👃 😃 , so I refactored into a delegate.